http-server-microservice-provider/src/main/java/io/silverware/microservices/providers/http/HttpServerMicroserviceProvider.java

```diff
@@ -67,7 +67,7 @@
    public void initialize(final Context context) {
       this.context = context;
       context.getProperties().putIfAbsent(HTTP_SERVER_PORT, 8080);
-      context.getProperties().putIfAbsent(HTTP_SERVER_ADDRESS, "localhost");
+      context.getProperties().putIfAbsent(HTTP_SERVER_ADDRESS, "0.0.0.0");
```

Comment:
Why is `0.0.0.0` the default server address? I would expect either `localhost` or `127.0.0.1`.